### ISSUE TITLE
Fixing the demo (Accepting updates?)

### ISSUE THREAD

**Issue by michaelwooley** [grimmer0125/webpack-pyodide-demo#5]
Hi, in order to get the example to work, I made several changes and collected them in a branch.

The branch is [here](https://github.com/michaelwooley/pyodide-webpack-example/tree/michael/fix-setup) ([diff relative to main](https://github.com/pyodide/pyodide-webpack-example/compare/main...michaelwooley:michael/fix-setup)).

Since the changes are fairly extensive, I don't know whether there would be any interest in a PR? (Also open to the possibility that the demo doesn't work for me because I failed to follow the instructions fully or my environment is somehow different.)

**Comment by rth** [comment]
Yes, definitely please open a PR. Ideally, it would be also good to make CI run https://github.com/pyodide/pyodide-webpack-example/pull/4. Thanks @michaelwooley !

**Comment by PhucNgo1711** [comment]
@michaelwooley's PR finally resolved my 3-day struggles!
